src/components/common/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,57 @@
+import { H4 } from '@/components/common/typography';
+import clsx from 'clsx';
+import { PropsWithChildren, useEffect, useRef } from 'react';
+import * as styles from './Accordion.css';
+import Image from 'next/image';
+import IconArrowDown from '/public/icons/Arrow-Down.svg';
+
+interface AccordionProps extends PropsWithChildren {
+  open: boolean;
+  title: string;
+  titleSuffix?: React.ReactElement;
+  onClick: () => void;
+}
+
+const Accordion: React.FC<AccordionProps> = ({
+  open,
+  title,
+  titleSuffix,
+  onClick,
+  children
+}) => {
+  const bodyWrapperRef = useRef<HTMLDivElement>(null);
+  const bodyRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!bodyWrapperRef.current || !bodyRef.current) return;
+    const height = open ? bodyRef.current.offsetHeight + 'px' : '0px';
+    bodyWrapperRef.current.style.height = height;
+  }, [open]);
+
+  return (
+    <div className="accordion">
+      <div className={styles.header} onClick={onClick}>
```

Comment:
아코디언 열고 닫을 수 있는게 아코디언 컴포넌트의 기본 동작인것 같은데 외부에서 상태를 전달받아서 쓰게되면, 아코디언 컴포넌트를 사용할때마다 open 과 관련된 useState를 선언해주어야 할것같아요!!
내부에 state를 선언하고, 

```suggestion
      const handleClick = (e) => {
      onClick(e);
      setOpen(prev=>!prev)
      }
      ...
      <div className={styles.header} onClick={handleClick}>
```

과 같이 click할 때마다 상태를 바꿔주는게 어떨까요?

추가로, 아코디언 open / close 외의 동작이 없다면 onClick props를 받지 않아도 될것같습니다!

Reply:
너무너무 좋은 의견입니다! 반영할게요 감사합니다😊